packages/cdktf-cli/templates/python/cdktf.json

```diff
@@ -1,6 +1,6 @@
 {
   "language": "python",
-  "app": "pipenv run ./main.py",
+  "app": "pipenv run python main.py",
```

Comment:
That's cross platform compatible?

Reply:
It worked in the two platforms I tested on.
On *nix systems the shebang line allows for switching the interpreter. On Windows that doesn't work, so explicit specification is generally necessary.